application/src/main/kotlin/net/corda/v5/application/flows/Subflow.kt

```diff
@@ -0,0 +1,24 @@
+package net.corda.v5.application.flows
+
+import net.corda.v5.base.annotations.Suspendable
+import net.corda.v5.base.exceptions.CordaRuntimeException
+
+/**
+ * A flow that should be started by another flow.
+ *
+ * When a flow implementing [Subflow] is called via the subflow API (see [FlowEngine]), the [call] method of this
+ * interface will be invoked.
+ */
+interface Subflow<out T> : Flow {
```

Comment:
Is `Subflow` or `SubFlow` the correct spelling 🤔 honest question.

Reply:
`FlowEngine.subFlow` is spelt with a capital `F`. We should choose one of them at least, I think `SubFlow` is probably correct.

Reply:
Yeah we're going to need to be consistent here! I prefer lowercase personally, but seeing as it's already uppercase everywhere I think we're sort of settled on that variant, so I'll change this.